### ISSUE TITLE
How to get cipher description for all supported ciphers?

### ISSUE THREAD

**Issue by dylc5190** [radiator-software/p5-net-ssleay#211]
I understand that Net::SSLeay::CIPHER_description($cipher, $buf, $size) can get the description of a cipher. But how can I come out the input $cipher from the results of Net::SSLeay::get_cipher_list? I suppose there should be a wrapper function for SSL_CTX_get_ciphers.

**Comment by dylc5190** [comment]
This works for me.
```
void
SSL_get_ciphers(s)
        SSL *              s
    PREINIT:
        STACK_OF(SSL_CIPHER) *sk = NULL;
        SSL_CIPHER *c;
       int i;
    PPCODE:
       sk = SSL_get_ciphers(s);
       if( sk == NULL ) {
           XSRETURN_EMPTY;
       }
       for (i=0; i<sk_SSL_CIPHER_num(sk); i++) {
           c = sk_SSL_CIPHER_value(sk, i);
           XPUSHs(sv_2mortal(newSViv(PTR2IV(c))));
       }
```

**Comment by h-vn** [comment]
This is now included in pull request #253
